README.md

```diff
@@ -28,9 +29,9 @@ sudo bash scripts/patch-etc-hosts.sh
 
 ## Getting Started
 1. Install requirements listed above.
-2. Setup /etc/hosts entries listed above. 
-3. Clone the repository, and open a terminal in the base of this project.
-4. Run the command `npm install` to install all the dependencies.
+1. Setup `/etc/hosts` entries listed above.
+1. Clone the repository, and open a terminal in the base of this project.
+1. Run the command `npm install` to install all the dependencies.
```

Comment:
Please fix the numbering

Reply:
This is Markdown syntax for on ordered list. The correct numbers are used when the page is rendered. Makes reordering steps much less of a hassle.

Reply:
I'm happy to go back to using hard coded numbers, though.